src/index.html

```diff
@@ -11,6 +11,51 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="get"
+      class="form"
+      data-qa="big"
+    >
+      <label class="form__label">
+        <div class="form__input-wrapper">
+          <input
+          type="text"
+          class="form__input"
+          placeholder="Try &quot;Los Angeles&quot;"
+          data-qa="keypress"
+          >
+
+          <img
+          src="./images/Search.svg"
+          alt="Search icon"
+          class="form__image"
+          >
```

Comment:
[Recommendation]: Create it using `background-image` property, because this image is a decorative one